source/tyr/tyr/resources.py

```diff
@@ -298,6 +297,24 @@ def put(self, id=None, name=None):
         parser.add_argument('import_ntfs_in_mimir', type=inputs.boolean,
                             help='import ntfs data in global autocomplete',
                             location=('json', 'values'), default=instance.import_ntfs_in_mimir)
+        parser.add_argument('min_nb_journeys', type=int,
+                            help='minimum number of different suggested journeys', location=('json', 'values'),
+                            default=instance.min_nb_journeys)
+        parser.add_argument('max_nb_journeys', type=int, required=False,
+                            help='maximum number of different suggested journeys', location=('json', 'values'))
+        parser.add_argument('min_journeys_calls', type=int,
+                            help='minimum number of calls to kraken', location=('json', 'values'),
```

Comment:
Why is there a parameter to set this? By setting the 'min_nb_journeys', the number of calls to kraken doesn't really matter, does it?

Reply:
it's more a max_extra_journey_call as we want to force more than one call to kraken, except that we don't want to call kraken a third time if we have called it two times. 
Hope it helps understanding the feature...